packages/gatsby-remark-embed-snippet/src/utils/get-language.js

```diff
@@ -0,0 +1,30 @@
+"use strict"
```

Comment:
```suggestion
```

let's not use this in source files

Reply:
The `index.js` of this plugin has this line. Remove them in a different PR if required?